api-operator/pkg/mgw/deployment.go

```diff
@@ -91,6 +90,23 @@ func Deployment(api *wso2v1alpha1.API, controlConfigData map[string]string, owne
 		corev1.ResourceCPU:    resource.MustParse(resLimitCPU),
 		corev1.ResourceMemory: resource.MustParse(resLimitMemory),
 	}
+
+	// container ports
+	containerPorts := []corev1.ContainerPort{
```

Comment:
what is the standard here?
`[]corev1.ContainerPort{`
or 
`[]corev1.ContainerPort {`